modules/terraform-aci-interface-state/variables.tf

```diff
@@ -0,0 +1,50 @@
+variable "pod_id" {
+  description = "Interface Pod ID. Minimum value: `1`. Maximum value: `255`."
+  type        = number
+  default     = 1
+
+  validation {
+    condition     = var.pod_id >= 1 && var.pod_id <= 255
+    error_message = "Minimum value: `1`. Maximum value: `255`."
+  }
+}
+
+variable "node_id" {
+  description = "Interface Node ID. Minimum value: `1`. Maximum value: `4000`."
+  type        = number
+
+  validation {
+    condition     = var.node_id >= 1 && var.node_id <= 4000
+    error_message = "Minimum value: `1`. Maximum value: `4000`."
+  }
+}
```

Comment:
Minimum node ID should be 101 since we can only blacklist switch interfaces.

Reply:
Fixed this.